api/Entities/ImageEntity.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace iTechArt.Hotels.Api.Entities
+{
+    public class ImageEntity
+    {
+        public int HotelId { get; set; }
+        [Key]
+        [MaxLength(58)]
```

Comment:
Why did you choose 58?

Reply:
It is folder + guid (36) + extension (4 or 5)